.github/workflows/mariadb.yml

```diff
@@ -57,11 +70,14 @@ jobs:
         # To list available plugins, you can run:
         #   cmake -L $MARIADB_SOURCE_DIR | grep PLUGIN_
         run: |
+          ccache --show-stats --verbose --version || :
```

Comment:
Could you do this in separated step for easy to confirm?

Reply:
fix: [1967479](https://github.com/mroonga/mroonga/pull/901/commits/196747998df752859d2cf9dea940552e675664f4) Sure.